frescobaldi_app/quickinsert/spanners.py

```diff
@@ -296,6 +300,107 @@ def spanner_positions(cursor):
     return positions
 
 
+def melisma_positions(cursor: QTextCursor) -> list:
+    """ Return the correct positions of the melisma commands, after all spanners already attached to the note """
+
+    def search_spanners_symbols(item1, item2, spanners) -> int:
+        """Find the spanners in the list lying between two music items.
+        :param item1: First item
+        :param item2: Second item
+        :param spanners: A list of spanners to be examined
+        :return: The position in the document of the last spanner attached to the first item, or None if nothing is found
+        """
+
```

Comment:
nitpick: (I know that we are not exactly following PEP style guides on every line of the code. But for new additions I will try to remark on stuff that I observe) - "There's no blank line either before or after the docstring."